doc/server/hap2/how-to-setup-livestatus.md

```diff
@@ -0,0 +1,105 @@
+# How to setup Livestatus API
+-----------------------------------------------------------------------------------
+
+You should install livestatus module, if you use hap2_nagios_livestatus.py plugin.
+
+This document describe for how to install livestatus module on CentOS 7 and Ubuntu 14.04 LTS.
+
+This document is written with the assumption that Nagios is installed.
+
+## (1) Install the livestatus module package
+
+### CentOS 7
+
+    # yum install epel-release
+    # yum install check-mk-livestatus 
+
+### Ubuntu 14.04 LTS
+
+    # apt-get install check-mk-livestatus
+
+## (2) Check livestauts library path
+
+You execute the following command on each OS to check the path of "livestatus.o".
+
+The path is replaced the next section's LIVESTATUS_LIB_PATH.
+
+### CentOS 7
+
+    # rpm -ql check-mk-livestatus
+
+### Ubuntu
+
+    # dpkg -L check-mk-livestatus
+
+## (3) Edit /etc/nagios/nagios.cfg to use an livestatus broker module
+
+Add the following line to /etc/nagios/nagios.cfg.
+
+Livestatus module make AF UNIX socket file.
```

Comment:
make -> makes
or
creates

And in this context 'file' is countable.
AF UNIX socket file ->
an AF UNIX socket file